assets/large_language_models/rag/components/src/validate_deployments.py

```diff
@@ -404,7 +403,96 @@ def main(parser_args, activity_logger: Logger):
         json.dump({"deployment_validation_success": "true"}, f)
 
     activity_logger.info(
-        "[Validate Deployments]: Success! Deployments have been validated.")
+        "[Validate Deployments]: Success! AOAI deployments have been validated.")
+
+
+def get_openai_model(model, key, activity_logger: Logger):
+    """Get model info from OpenAI"""
+    endpoint = f"https://api.openai.com/v1/models/{model}"
```

Comment:
Calling openai APIs directly instead of using openai package because the package gets tied to AOAI credentials?

Reply:
Yes, and it uses auth to see if you have access to the model this way as well